iridauploader/tests_integration/integration_data_setup.py

```diff
@@ -83,6 +81,9 @@ def __init__(self, base_url, user, password, branch, db_host, db_port):
         self.REPO_PATH = path.join(self.PATH_TO_MODULE, "repos")
         self.IRIDA_PATH = path.join(self.REPO_PATH, "irida")
 
+        # Handles the irida subprocess
+        self.irida_subprocess = None
```

Comment:
this is just initializing the attribute so that you can use it in `install_irida`?

Reply:
I init it there to be used in `run_irida` and `stop_irida`, In python it's good practice to init an attribute to `None` in the `__init__` when it's used in multiple functions. You can also do it as a class property, but depending on how it's used the IDE complains sometimes.